connector-commons/http-client/src/test/java/io/camunda/connector/http/client/TestDocumentFactory.java

```diff
@@ -39,9 +41,43 @@ public Document resolve(DocumentReference reference) {
   @Override
   public Document create(DocumentCreationRequest request) {
     var reference = store.createDocument(request);
-    var metadata = new DocumentMetadata();
-    metadata.setContentType(reference.getMetadata().getContentType());
-    metadata.setFileName(reference.getMetadata().getFileName());
+    var metadata =
```

Comment:
Why this change?

Reply:
Side effect of switching to an implementation in our SDK instead of reusing the one from the camunda client. Good finding, cleaned it up